Chawki-Mbarki/Body-Building
# Desktop

# Super Power Fitness Center Website
👋 Hi there,

This pull request contains the code for the Super Power Fitness Center website, a responsive and dynamic website for a gym.

## What I did
- I followed the given design and personalized the content for the gym.
- I made the website responsive for all screen sizes with a `width` of `320px` or higher.
- I generated the Trainers section dynamically using `JavaScript`.
- I added all necessary assets to match the theme of the website.
- I implemented functional animation to the navigation using `JavaScript` and `CSS`.
- Deployed the website using GitHub pages.
- Recorded a video explaining/showing different sections of the project.
## Challenges
- I tried to make the website look exactly the same as the design, but unfortunately, I couldn’t achieve that due to the lack of provided information and resources.
## How to test
To test the website, you can clone this branch and open the `index.html` file in your browser.
You can also use tools like Lighthouse or Webhint to check the performance and accessibility of the website.
## Feedback
I would appreciate any feedback or suggestions on how to improve the website or the code quality. Thank you for your time and attention. 😊
